src/WebJobs.Script/Binding/BindingContext.cs

```diff
@@ -12,7 +13,9 @@ public class BindingContext
 
         public object Input { get; set; }
 
-        public Stream Value { get; set; }
+        public object Value { get; set; }
```

Comment:
hmmm, perhaps you haven't rebased? Master already contains my change to make Value of type object.